backend/resolvers/mutations/entry.js

```diff
@@ -28,17 +29,47 @@ const createEntry = (entry, entryType, entryId, t) => {
       // property and replace it with the group's ID, since our orm recognizes
       // groupId, not a Group
 
+      let userFindPromise = Promise.resolve(null)
+      if (entry.studentUsername){
+        userFindPromise = User.findById(entry.studentUsername);
+      }
+
       // clone the object
       let newEntry = Object.assign({}, entry)
       // remove the 'group' property
       delete newEntry['group']
       // create the new Entry with select properties filled-in
-      return Entry.create({
-        ...newEntry,
-        entryType: entryType,
-        entryId: entryId,
-        groupId: group ? group.id : null
-      }, { transaction: t })
+      return userFindPromise.then(user => {
+
+        let userUpdatePromise = Promise.resolve(null)
+        if (user.hometown != entry.hometown){
+          userUpdatePromise = User.update(
+            {
+              hometown: entry.hometown
+            },
+            { where: {username: entry.studentUsername}}
+          )
+        }
+
+        if (user.displayName != entry.displayName){
+          userUpdatePromise = User.update(
+            {
+              displayName: entry.displayName
+            },
+            { where: {username: entry.studentUsername}}
+          )
+        }
+        delete newEntry['hometown'];
+        delete newEntry['displayName'];
```

Comment:
I don't think this needs to get deleted.

Reply:
addresed